Custom annotations 329 (#657)
zalando/postgres-operator: docs/reference/cluster_manifest.md

   then the default priority class is taken. The priority class itself must be
   defined in advance. Optional.

* **podAnnotations**

>   This map of strings gets attached as an annotation to each pod created for the database.

Reviewer: ```suggestion
  A map of key value pairs that gets attached as [annotations](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/)
  to each pod created for the database.
```
